### ISSUE TITLE
how can i use only one subgraph?

### ISSUE THREAD

**Issue by zsgj-Xxx** [ZikangZhou/HiVT#26]
I masked other vehicles, made sure only one moving vehicle was left in each scene, and tried to predict the track, which resulted in my edge_ Index is an empty list and cannot continue. What is a good solution?

**Comment by ZikangZhou** [comment]
Hi, 

Thanks for your interest. You can write something like this:
```
edge_index = torch.tensor([[], []], dtype=torch.long)
```

This will create an edge index tensor with the shape of [2, 0], and then you can pass this tensor to the model in the forward pass.

**Comment by zsgj-Xxx** [comment]
Thank you for your quick response!

You're right, I've got the results.
